admin/class-yoast-form.php

```diff
@@ -77,9 +77,9 @@ public function admin_header( $form = true, $option = 'wpseo', $contains_files =
 		if ( $form === true ) {
 			$enctype = ( $contains_files ) ? ' enctype="multipart/form-data"' : '';
 
-			if ( Yoast_Network_Settings_API::meets_requirements() ) {
+			if ( Yoast_Network_Admin::meets_requirements() ) {
```

Comment:
I prefer doing: 
```
$network_admin = new Yoast_Network_Admin();
if ( $network_admin->meets_requirements() ) {
    $action_url       = network_admin_url( 'settings.php' );
    $hidden_fields_cb = array( network_admin, 'settings_fields' );
}
```

Reply:
See my previous comment regarding the singleton principle.